### PR TITLE
deps: update to webpack 4

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
The changes here look fine, and lib-jitsi-meet builds fine, but jitsi-meet has an error while building the native app. Is it a conflict with webpack in jitsi-meet?

```Unable to resolve module `lib-jitsi-meet/lib-jitsi-meet.min` from `/home/jenkins/jitsi-meet/react/features/base/lib-jitsi-meet/_.native.js`: Module `lib-jitsi-meet/lib-jitsi-meet.min` does not exist in the Haste module map```

**Comment by saghul** [comment]
Thanks, I'll take a look. I think I have an idea of what's wrong, note the change I had to make in the Makefile...

**Comment by virtuacoplenny** [comment]
Jenkins test this please.

File: webpack.config.js [hunk 1]
```diff
@@ -1,31 +1,14 @@
...
-        cache: true,
-        extractComments: true,
-        parallel: true,
-        sourceMap: true
-    }));
-}
 
 const config = {
     devtool: 'source-map',
+    mode: minimize ? 'development' : 'production',
```
Comment (hristoterezov) [inline webpack.config.js]: @saghul @virtuacoplenny Shouldn't this be exactly the opposite? I think at the moment when you pass -p we will be in development mode.

Comment (saghul) [reply webpack.config.js]: Looks like I am, indeed and idiot. You're totally right!

**Pull request merged**